File: solutions/python/VB_T1_010_double_or_nothing.py
"""VB-T1-010: Double or Nothing -- Python baseline."""
def double_or_nothing(x: int) -> int:
    return x * 2 if x > 0 else 0

if __name__ == "__main__":
    assert double_or_nothing(5) == 10
    assert double_or_nothing(0) == 0
    assert double_or_nothing(-3) == 0
    print("VB-T1-010 ok")

File: solutions/typescript/VB_T1_010_double_or_nothing.ts
// VB-T1-010: Double or Nothing -- TypeScript baseline
function doubleOrNothing(x: number): number { return x > 0 ? x * 2 : 0; }
console.assert(doubleOrNothing(5) === 10);
console.assert(doubleOrNothing(-3) === 0);
